src/main/java/io/vertx/spi/cluster/hazelcast/impl/Throttling.java

```diff
@@ -0,0 +1,114 @@
+/*
+ * Copyright 2021 Red Hat, Inc.
```

Comment:
I'd use the "Copyrights contributors to the Eclipse Foundation" as in other files

Reply:
We have used this header for files in projects that do not belong to `eclipse-vertx` org.